RELEASE-NOTES.md

```diff
@@ -11,29 +11,17 @@
   Gaussian Process implementation for efficient inference, along with
   lower-level functions such as `cartesian` and `kronecker` products.
 - Added `Coregion` covariance function.
-- Add new 'pairplot' function, for plotting scatter or hexbin matrices of sampled parameters.
-  Optionally it can plot divergences.
+- Add new 'pairplot' function, for plotting scatter or hexbin matrices of sampled parameters. Optionally it can plot divergences.
 - Plots of discrete distributions in the docstrings
 - Add logitnormal distribution
+- New function `pm.sample_prior` which generates test data from a model in the absence of data.
```

Comment:
I would move this further up as it's a major feature. Also, I think we should add author names to who contributed to feature / bugfix.